client/src/components/ClassCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Card, Button } from 'semantic-ui-react';
+
+import { putUserClass } from '../utils/apiWrapper';
+
+// import '../css/Class.scss';
+
+function ClassCard(props) {
+  const { text } = props;
+  return (
+    <Card style={{ height: '85px' }}>
```

Comment:
For best practices, you should never have inline styling. Extract styles to a separate scss file